edx_repo_tools/codemods/django42/github_actions_modernizer_django42.py

```diff
@@ -0,0 +1,52 @@
+"""
+Modernizer for Github Actions CI
+"""
+from copy import deepcopy
+import click
+from edx_repo_tools.utils import YamlLoader
+
+ALLOWED_DJANGO_VERSIONS = ['django32', 'django40', 'django42']
```

Comment:
Let's add `django41` in the list as well to avoid removing `django41` if it is already added in any repo.
```suggestion
ALLOWED_DJANGO_VERSIONS = ['django32', 'django40', 'django41', 'django42']
```

Reply:
ok